### DISCUSSION TITLE
Wazuh vulnerability scanning & monthly rollups

### DISCUSSION THREAD

**Discussion by Jaap79** [Security-Onion-Solutions/securityonion#8670] [Unsupported Versions] [answered]
Hi,

Something we're coming across in SO 2.3.150 (but also in previous version .130):

I set up Wazuh for vulnerability scanning by browsing through the Discussions site here. It all seems to work just fine with the offline NVD database being populated and all. No problems on that end.

However, we get dozens (nay, hundreds!) of CVE's missing on servers. 

Say, for instance, we're talking about a Windows box called Server001 and CVE-2022-1234 (just an example number).

- CVE-2022-1234 Missing on Server001
- CVE has Microsoft KB1234567 as security-only fix
- The CVE was fixed in the Monthly Rollup Update of January 2022 with KB2345678

Technically, the KB1234567 is indeed missing, because factually it was not installed seperately. Powershell confirms this when I check for installed updates. However, it was part of the Monthly Rollup Update of January and since those are cumulative, it will still be fixed in the Monthy Rollup Update for, say, August 2022.

When I check the Server001 for installed updates, I see that the Monthly Rollup August 2022 is currently installed (KB4567890 for instance) so the server is not actually vulnerable to this CVE. However, Wazuh does not recognize any of the Monthly Rollups as containing KB1234567 ... and therefore triggers the alarm each time it scans.

What is the best approach for this? Or rather, what real-world applications do you have for Vulnerability Scanning in Wazuh related to SO ?

**Comment by dougburks** [comment, marked as answer]
These kinds of issues can happen with the vulnerability scanning in Wazuh and that's why we don't enable that feature by default.
